### ISSUE TITLE
Difficulties installing JupyterLab-Vim

### ISSUE THREAD

**Issue by Dcosthephalump** [Accessible-Data-and-Code/jupyterlab-ide#55]
I was going to start the next notebook with a blank slate environment to make sure that I had everything working. I removed the jupyterlab-ide environment. When I attempted this, I started to get this error repeatedly:

```console
InvalidArchiveError('Error with archive C:\\Users\\David\\anaconda3\\pkgs\\jupyterlab-vim-v0.8.0-0.tar.bz2.  You probably need to delete and re-download or re-create this file.  Message was:\n\nfailed with error: seeking backwards is not allowed')
```

I had a similar issue [here](https://github.com/nicole-brewer/jupyterlab-ide/issues/6#issuecomment-1591774143).

I tried uninstalling and reinstalling Anaconda. This worked last time I had a similar issue. It did not work this time.

I have temporarily solved this problem by locally removing the ```jupyterlab-vim``` package from the environment.yml and installing it after building the environment. The problem does not exist for the Binder, although Binder does not seem to allow for ```jupyterlab-vim``` to be used when it creates an environment.

Open to solutions. Maybe it's a Windows thing?

**Comment by nicole-brewer** [comment]
1) Maybe we look up what version is installed when we install "after-the-fact". Then specify that version that worked. 

2) Can you add 
```
dependencies:
- conda
  - jupyterlab-vim
```
to force conda to run that command separately and after the fact? Hopefully?

3) If that doesn't work,  put the "conda install... " line in a [postBuild script](https://mybinder.readthedocs.io/en/latest/using/config_files.html#postbuild-run-code-after-installing-the-environment). IF this is the route we take, we need to update REAME.md with further instructions

**Comment by Dcosthephalump** [comment]
1. This did not work. I got a package not found error when I tried to install with the specific package number that worked after the fact.

2. This returned the same error as solution 1.

3. For [code formatter](https://anaconda.org/conda-forge/jupyterlab_code_formatter) and for [vim](https://anaconda.org/conda-forge/jupyterlab_vim), I followed the conda-forge instructions after installation and got the desired results.

4. I did try making jupyterlab-vim and jupyterlab-code-formatter ```pip``` installed dependencies. This worked without problems.

I see two options:
  1. Use ```pip``` for installing code formatter and vim
  2. Go with the postBuild script. I prefer ```pip```.
